charms/kserve-controller/metadata.yaml

```diff
@@ -8,13 +8,11 @@ description: |
   KServe Controller.
 summary: |
   KServe Controller.
-
```

Comment:
Do we want to add doc links here or should we do it in separate PR?

Reply:
Good catch, but we don't have the kserve docs in [the bundle](https://charmed-kubeflow.io/docs/kubeflow-bundle). I'll submit a new patch for that.

Reply:
Great!